exonum-java-binding-time-service/src/main/java/com/exonum/binding/time/TimeSchemaImpl.java

```diff
@@ -24,22 +24,28 @@
 import com.exonum.binding.storage.indices.ProofMapIndexProxy;
 import java.time.ZonedDateTime;
 
-public class TimeSchemaProxy implements TimeSchema {
+public class TimeSchemaImpl implements TimeSchema {
```

Comment:
IMO, the previous (concrete) name was better than generic 'impl' which gives zero clue about the class.